viscm/bezierbuilder/curve.py

```diff
@@ -0,0 +1,59 @@
+from math import factorial
+
+import numpy as np
+
+
+def binom(n, k):
+    """Re-implement `scipy.special.binom`.
+
+    Reimplementing it ourself lets us avoid pulling in a dependency scipy just for that
+    one function.
+
+    FIXME: `scipy` is already a dependency. Delete this?
+    """
+    return factorial(n) * 1.0 / (factorial(k) * factorial(n - k))
+
+
+def Bernstein(n, k):
+    """Bernstein polynomial."""
+    coeff = binom(n, k)
+
+    def _bpoly(x):
+        return coeff * x**k * (1 - x) ** (n - k)
+
+    return _bpoly
+
+
+def Bezier(points, at):
+    """Build Bézier curve from points.
+    Deprecated. CatmulClark builds nicer splines
+    """
+    at = np.asarray(at)
```

Comment:
Should this raise a FutureWarning to indicate the deprecation?

Reply:
ea93bc2